### PR TITLE
feat: switch to bitnami repository

### LINKED ISSUE #118: Update chart repository URL to Bitnami
Issue #115 fixed the issue with the Helm "stable" repo URL move, but all of the most recent charts minibroker uses in that repo are marked as DEPRECATED by the Bitnami maintainers. It may be time to simply point to the Bitnami repo for the latest charts.

I've tested this by setting `helmRepoUrl: "https://charts.bitnami.com/bitnami"` and deploying the latest releases of RabbitMQ, PostgreSQL, MariaDB, MongoDB, and Redis. I have not tested MySQL, but I assume this works as well.

### PR DESCRIPTION
**Pull request by f0rmiga**
Fixes #118.

### CODE REVIEW

File: pkg/minibroker/minibroker.go [hunk 1]
```diff
@@ -108,20 +127,6 @@ func NewClient(
 	}
 }
 
-func loadInClusterClient() kubernetes.Interface {
```
Comment (mook-as) [inline pkg/minibroker/minibroker.go:111]: FWIW, we could have passed both clients back instead. (Not an issue to inline it, of course.)

File: pkg/minibroker/minibroker.go [hunk 2]
```diff
@@ -389,29 +394,48 @@ func (c *Client) provisionSynchronously(instanceID, namespace, serviceID, planID
...
-		err := c.labelSecret(secret, instanceID)
+
+		var dr dynamic.ResourceInterface
+		if r.Mapping.Scope.Name() == meta.RESTScopeNameNamespace {
+			// Namespaced resources.
+			dr = c.dynamicClient.Resource(r.Mapping.Resource).Namespace(obj.GetNamespace())
+		} else {
+			// Cluster-wide resources.
+			dr = c.dynamicClient.Resource(r.Mapping.Resource)
+		}
```
Comment (mook-as) [inline pkg/minibroker/minibroker.go:428]: Does something like this work?
```suggestion
		dr := c.dynamicClient.Resource(r.Mapping.Resource)
		if r.Mapping.Scope.Name() == meta.RESTScopeNameNamespace {
			// Namespaced resources.
			dr = dr.Namespace(obj.GetNamespace())
		}
```

File: pkg/minibroker/minibroker.go [hunk 3]
```diff
@@ -570,20 +536,20 @@ func (c *Client) bindSynchronously(
 			Services(releaseNamespace).
 			List(ctx, filterByInstance)
 		if err != nil {
-			return err
+			return errors.Wrapf(err, "failed to get services")
 		}
 		if len(services.Items) == 0 {
-			return osb.HTTPStatusCodeError{StatusCode: http.StatusNotFound}
+			return fmt.Errorf("failed to get services: no services found")
```
Comment (mook-as) [inline pkg/minibroker/minibroker.go:542]: I thought we needed to return the OSB-specific errors because the framework handle them specially?

**Review by mook-as — CHANGES_REQUESTED** [review]

**Comment by mrbobbytables** [comment]
This project is being archived, closing open issues and PRs.
Please see this PR for more information: https://github.com/kubernetes/community/pull/6632

**Pull request closed without merging**